spec/javascripts/indigenousLanguagesLayers_spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Testing for indigenous Language layer", function() {
+
+  beforeEach(function () {
+      loadFixtures('indigenousLanguagesLayer.html');
+    });
+
+  it("Basic Test", function () {
+  expect(true).toBe(true);
+  });
+
+  it("Checks if layer polygons are present", function () {
+  // There should be atleast one polygon in the Language layer always .
+  expect($("#map").children()[0].childNodes[2].childNodes[0].childNodes[0].childNodes.length).toBeGreaterThan(1);
```

Comment:
Hi...this is correct testing .
But you have to understand that these rectangles are made from a remote API, hence it takes time for these rectangles to appear on page, right?

So there are two things you should try in the following order:
1. Try to preempt the execution of test till the rectangles appear on website . 
https://jasmine.github.io/tutorials/async

2. If the asynchronous approach of testing does not works for some reason, what we can do is to look at what we have to actually test i.e the layer should be loaded when the respective function is called , right? 
So what we can do is basically in the fixture add a L.polygon object on the map and test if it occurs on map .
In this way atleast we can test that the layer is being loaded or not . Later we can test if the actual data is getting loaded from remote API or not . Makes sense?

cc @jywarren

Reply:
Thanks,
I'll give this another try and get back to you.

Reply:
The async methods don't seem to work.
I'll try the 2nd method